kolibri/core/assets/src/state/modules/session.js

```diff
@@ -72,6 +74,9 @@ export default {
   },
   mutations: {
     CORE_SET_SESSION(state, value) {
+      const serverTime = value.server_time;
+      setServerTime(serverTime);
```

Comment:
I don't think we should call functions with non-vuex side effects inside mutations. Maybe we need a `setSession` action?

Reply:
Have addressed this in latest commits.